playbooks/group_vars/all.yml

```diff
@@ -0,0 +1,49 @@
+---
+upstream_repos_clone_folder: "/opt"
+suse_osh_registry_location: "{{ lookup('env','OPENSTACK_HELM_IMAGE_REGISTRY_LOCATION') | default('docker.io', true) }}"
+suse_osh_image_version: "{{ lookup('env','OPENSTACK_HELM_IMAGE_VERSION') | default('pike', true) }}"
+suse_libvirt_image_version: "{{ lookup('env','OPENSTACK_HELM_LIBVIRT_IMAGE_VERSION') | default('ubuntu-xenial-1.3.1-1ubuntu10.24', true) }}"
+suse_ovs_image_version: "{{ lookup('env','OPENSTACK_HELM_OPENVSWITCH_IMAGE_VERSION') | default('v2.8.1', true) }}"
+
+developer_mode: "{{ (lookup('env','SOCOK8S_DEVELOPER_MODE') | default('False', true) ) | bool }}"
```

Comment:
already defined in common vars.

Reply:
yes..so intention is to replace common-vars with **group_vars/{group_of_hosts}.yml** of sharing variables across group of hosts.
So whatever is defined under common-vars is current places in group_vars/all.yml for the reasons mentioned above.